lib/trip_dispatcher.rb

```diff
@@ -27,14 +34,38 @@ def inspect
               #{passengers.count} passengers>"
     end
 
+    def request_trip(passenger_id)
+      passenger = find_passenger(passenger_id)
+      driver = @drivers.select {|driver| driver.status == :AVAILABLE}.first
```

Comment:
You could make this clearer using `find`:

```suggestion
      driver = @drivers.find {|driver| driver.status == :AVAILABLE}
```